### PR TITLE
Docs misc 14

### PR DESCRIPTION
**Pull request by smoe**

### CODE REVIEW

File: docs/src/gui/qtvcp-development.adoc [hunk 1]
```diff
@@ -171,10 +171,10 @@ After this function is called the widget has access to some special
...
-*`self.HAL_NAME`*:: This _widget's name_ as a string
-*`self.QT_OBJECT_`*:: This _widget's PyQt object instance_
-*`self.QTVCP_INSTANCE_`*:: The _very top level parent_ of the screen
-*`self.PATHS_`*:: The _instance of QtVCP's path_ library
+*`self.HAL_NAME`*:: this _widget's name_ as a string
+*`self.QT_OBJECT_`*:: this _widget's PyQt object instance_
+*`self.QTVCP_INSTANCE_`*:: the _very top level parent_ of the screen
+*`self.PATHS_`*:: the _instance of QtVCP's path_ library
 *`self.PREFS_`*:: the _instance of an optional preference file_
 *`self.SETTINGS_`*:: the `Qsettings` _object_
```
Comment (hansu) [inline docs/src/gui/qtvcp-development.adoc:179]: Why not all beginning with capital T?

Comment (smoe) [reply docs/src/gui/qtvcp-development.adoc:179]: The article is not needed in the first place IMHO. Did not want to do too much in that patch, I think.

Comment (smoe) [reply docs/src/gui/qtvcp-development.adoc:179]: Shall I rebase and remove the articles at that opportunity?

Comment (hansu) [reply docs/src/gui/qtvcp-development.adoc:179]: Look at https://linuxcnc.org/docs/2.9/html/gui/qtvcp-development.html#_widget_special_setup_functions
and decide which looks better to you - you'll find both "This" and "this" mixed in this section.
But it's more used with capital letter like in "4. Path Information"

Comment (smoe) [reply docs/src/gui/qtvcp-development.adoc:179]: Shall I then rebase and change to capitals? I truly do not care either way.

Comment (hansu) [reply docs/src/gui/qtvcp-development.adoc:179]: Yes :+1:

Comment (smoe) [reply docs/src/gui/qtvcp-development.adoc]: done

**Pull request merged**